third_party/2and3/flask/json/__init__.pyi

```diff
@@ -1,19 +1,19 @@
 # Stubs for flask.json (Python 3.6)
-#
-# NOTE: This dynamically typed stub was automatically generated by stubgen.
-
 import json as _json
 from typing import Any
+from jinja2 import Markup
 
 class JSONEncoder(_json.JSONEncoder):
     def default(self, o: Any): ...
 
 class JSONDecoder(_json.JSONDecoder): ...
 
+def detect_encoding(data: Any) -> str: ...
```

Comment:
Please mark as undocumented, since this is not in the API documentation. Also, `data` accepts only bytes.
```suggestion
def detect_encoding(data: bytes) -> str: ...  # undocumented
```